audiomentations/augmentations/shift.py

```diff
@@ -49,15 +49,14 @@ def __init__(
     def randomize_parameters(self, samples, sample_rate):
         super().randomize_parameters(samples, sample_rate)
         if self.parameters["should_apply"]:
-            self.parameters["num_places_to_shift"] = int(
-                round(
-                    random.uniform(self.min_fraction, self.max_fraction)
-                    * samples.shape[-1]
-                )
+            self.parameters["fraction_to_shift"] = random.uniform(
+                self.min_fraction, self.max_fraction
             )
 
     def apply(self, samples, sample_rate):
-        num_places_to_shift = self.parameters["num_places_to_shift"]
+        num_places_to_shift = round(
```

Comment:
int?

Reply:
If the argument `ndigits ` in the function `round ` is omitted, it returns an `int` already, that's why I removed the int()